setup.py

```diff
@@ -356,6 +363,9 @@
     'obspy.plugin.inventory.CSS': [
         'writeFormat = obspy.io.css.station:_write_css',
         ],
+    'obspy.plugin.inventory.SEISAN': [
+        'writeFormat = obspy.io.nordic.core:inventory_to_seisan',
```

Comment:
Mmm, I had a thought about that but forgot to mention it - happy to move it there - it's a slightly odd case in that it doesn't fit with the event things in `obspy.io.nordic`, nor really with the waveform things in `obspy.io.seisan`.  The format is probably more correctly called *STATION0.HYP* which is the file that seisan uses as input for *Hypocentre*.  I think either a name change to:
```python
   'obspy.plugin.inventory.STATION0': [...
```
or moving to the `obspy.io.seisan` would be worthwhile - any preference or other ideas?